.github/workflows/_wiki-documentation.yaml

```diff
@@ -31,7 +31,6 @@ permissions:
 jobs:
   publish-wiki:
     name: Publish Github Wiki
-    if: github.repository.has_wiki == true
```

Comment:
How about:
```suggestion
if: github.event.repository.has_wiki == true
```

If we remove the condition, would the release fail if the repo has no wiki?

Reply:
I’ve not tried it but would there be an `event` object (which contains a properly shaped `repository` object) in a reusable workflow?

Reply:
Works 👍🏼